### ISSUE TITLE
Drag & Drop: Add "AllowDrag" attached propeties

### ISSUE THREAD

**Issue by nazarenomanco** [RadicalFx/radical#153]
Adding a boolean "AllowDrag" attached propeties will allow to disable starting drag operation from some controls.

--

Depends on #154

**Comment by mauroservienti** [comment]
Depends on #154

**Comment by mauroservienti** [comment]
Closing as duplicate of #151
